models/ontology/taotest.rdf

```diff
@@ -95,7 +95,8 @@
   <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOTest.rdf#TranslationCompletionStatuses">
     <rdfs:label xml:lang="en-US"><![CDATA[Translation Statuses]]></rdfs:label>
     <rdfs:comment xml:lang="en-US"><![CDATA[Translation Statuses]]></rdfs:comment>
-    <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#SystemObject"/>
+    <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#List"/>
+    <generis:is_editable rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#False"/>
```

Comment:
I do not get why this property is special and needs this new `is_editable` attribute. It was never the case in TAO until now, why inventing this?